walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -65,12 +63,10 @@ else if (request.getAttendees().size() > 0) {
       if (optionalAttendees){
         if (noRequiredMembers && noOptionalMembers){
           continue;
-        }
+        }     
```

Comment:
nit: did you mean to add whitespace here?

Reply:
Thank you will fix that